src/components/BounceRuleDetailed/index.jsx

```diff
@@ -112,50 +121,135 @@ const BounceRuleDetailed = ({
         <Column width={10} offset={2}>
           <Changelog
             data-test="changelog-container"
-            handleButtonClicked={handleButtonClicked}
+            changelogLimit={changelogLimit}
+            handleChangelogClicked={handleChangelogClicked}
+            changelog={changelog}
+            isChangelogEmpty={isChangelogEmpty}
           />
         </Column>
       </Row>
-      <Row>
-        <Column width={4} offset={5}>
-          <Pagination
-            prevPageIndex={() => {}}
-            nextPageIndex={() => {}}
-            pageIndex={pageIndex}
-            pageInterval={pageInterval}
-            numRules={numRules}
-            updatePageIndex={() => {}}
-            pagesToDisplay={pagesToDisplay}
-          />
-        </Column>
-      </Row>
-
+      {!isChangelogEmpty && (
+        <Row>
+          <Column width={4} offset={5}>
+            <Pagination
+              prevPageIndex={1}
```

Comment:
Will this work if it expands out to be longer and we go to like 11 pages since it's hardcoded right now or will this pagination be handled in a separate ticket?

Reply:
will be handled in a seperate ticket